Wrappers.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+
+struct DishWasherWrapper
+{
+    DishWasherWrapper(DishWasher* d) : diWaPtr(d) {}
+    std::unique_ptr<DishWasher> diWaPtr = nullptr;
+}; 
+
+
+struct DishWashingProcessWrapper
+{
+    DishWashingProcessWrapper(DishWashingProcess* d) : washPtr(d) {}
+    std::unique_ptr<DishWashingProcess> washPtr = nullptr;
+}; 
+
+
+struct DrumMachineWrapper
+{
+    DrumMachineWrapper(DrumMachine* d) : drMaPtr(d) {}
+    std::unique_ptr<DrumMachine> drMaPtr = nullptr;
+}; 
+
+
+struct ServiceStationWrapper
+{
+    ServiceStationWrapper(ServiceStation* s) : serStaPtr(s) {}
+    std::unique_ptr<ServiceStation> serStaPtr = nullptr;
+}; 
+
+
+struct ProjectWrapper
+{
+    ProjectWrapper(Project* p) : prjPtr(p) {}
+    std::unique_ptr<Project> prjPtr = nullptr;
+}; 
```

Comment:
The first time you submitted Project5.3 I should have caught you using `std::unique_ptr`
While it is not wrong to use it (it is the proper tool for holding heap-allocated objects), the video demonstrated how you can manually manage heap-allocated objects yourself.
This is what you needed to do here.

<img width="741" alt="Screen Shot 2021-03-22 at 1 13 56 PM" src="https://user-images.githubusercontent.com/2929847/112052574-83171e00-8b10-11eb-83b4-e6f1bcbba74e.png">

All of the same requirements regarding NO IN-CLASS IMPLEMENTATIONS ALLOWED must be followed here as well
- [ ] revise: implement wrapper classes that demonstrate how to manage the memory themselves.
- [ ] revise: NO IN-CLASS IMPLEMENTATIONS ALLOWED